util/git.go

```diff
@@ -14,3 +15,25 @@ func UpdateGitRepo(ctx context.Context, gitpath string) {
 	Debugf(ctx, "%s: run %s\n", gitpath, cmd)
 	CheckCmd(cmd.CombinedOutput())
 }
+
+func IsPathIgnoredByGit(ctx context.Context, gitpath string, path string) bool {
```

Comment:
can you add a godoc comment here?
something like:

`// IsPathIgnoredByGit returns if a path is git ignored.`